frontend/src/slices/meditation/actions.ts

```diff
@@ -0,0 +1,29 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+import { type AsyncThunkConfig } from '#libs/types/types.js';
+import {
+  type MeditationEntryCreatePayload,
+  type MeditationEntryCreateResponseDto,
+} from '#packages/meditation/meditation.js';
+
+import { name as sliceName } from './meditation.slice.js';
+
+const createMeditationEntry = createAsyncThunk<
+  MeditationEntryCreateResponseDto,
+  MeditationEntryCreatePayload,
+  AsyncThunkConfig
+>(`${sliceName}/create-meditation-entry`, async (payload, { extra }) => {
+  const { meditationApi, filesApi, notification } = extra;
+
+  const file = await filesApi.uploadFile(payload.file);
+  const item = await meditationApi.createMeditationEntry({
+    mediaUrl: file.url,
+    contentType: file.contentType,
+  });
```

Comment:
we should send file in createMeditationEntry and upload it on backend
```suggestion
  const item = await meditationApi.createMeditationEntry(file);
```

Reply:
@Liza-Veis I asked about it before. Should I change this approach and make the only request and process the file to the url in the backend and not to use file api at all?

![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92298163/14d81ab3-f5f0-49fd-8ab8-c2d4f9c1167d)

Reply:
yes let's change it, we can either pass id or send file, not to allow to save any random url. but if we pass id, it's will be harder to validate file size and content type on backend

Reply:
Done